app/pages/api/getAchievements.js

```diff
@@ -0,0 +1,9 @@
+import { firestore, collection, doc, getDocs } from '@/utils/firebase'

```

Comment:
this function set's a collection reference so firebase can know what you're talking about then you ask it to give you the document it's called a snapshot afterwards you map through the document to get the data and assign it to a variable